featuremanagement/_featuremanager.py

```diff
@@ -154,107 +148,155 @@ def _variant_name_to_variant(self, feature_flag, variant_name):
                 return Variant(variant_reference.name, configuration)
         return None
 
-    def is_enabled(self, feature_flag_id, **kwargs):
+    @overload
+    def is_enabled(self, feature_flag_id, user_id, **kwargs):
         """
-        Determine if the feature flag is enabled for the given context
+        Determine if the feature flag is enabled for the given context.
 
-        :param str feature_flag_id: Name of the feature flag
-        :paramtype feature_flag_id: str
-        :return: True if the feature flag is enabled for the given context
+        :param str feature_flag_id: Name of the feature flag.
+        :param str user_id: User identifier.
+        :return: True if the feature flag is enabled for the given context.
         :rtype: bool
         """
-        return self._check_feature(feature_flag_id, **kwargs)["enabled"]
 
-    def get_variant(self, feature_flag_id, **kwargs):
+    def is_enabled(self, feature_flag_id, *args, **kwargs):
         """
-        Determine the variant for the given context
+        Determine if the feature flag is enabled for the given context.
 
-        :param str feature_flag_id: Name of the feature flag
-        :paramtype feature_flag_id: str
-        :return: Name of the variant
-        :rtype: str
+        :param str feature_flag_id: Name of the feature flag.
+        :return: True if the feature flag is enabled for the given context.
+        :rtype: bool
         """
-        return self._check_feature(feature_flag_id, **kwargs)["variant"]
+        targeting_context = TargetingContext()
```

Comment:
This is a nice way to offer both. Could pull this out into a helper since it's used in both is_enabled and get_variant